### ISSUE TITLE
Not sure if bug or not - Missing recipe book/button

### ISSUE THREAD

**Issue by Snorlax1337** [FTBTeam/FTB-Modpack-Issues#2427]
### Modpack

FTB Presents Direwolf20 1.19

### Is your suggestion related to a problem? Please describe.

So first of all: I am playing FTB-DW20 1.19.2 (or rather would like to)

...but...
I am looking for the mod/reason why my Recipe Book button is gone/hidden
I am talking about this function that enables the window on the left that tells me what i could craft with what i have in my inventory.

Is this even intentional? It is really hard for me to imagine that.

Someone told me it could be because of FastWorkbench.
So I have tried changing the settings of "FastWorkbench-1.19.2-7.0.1" and also removing it entirely, but it didnt help.
I also asked the dev of [Oh The Biomes You'll Go](https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go) about this, but all he said was, that this mod isnt at fault for hiding the recipe book button.

tl;dr
Somebody please tell me what i have to do, to get back my recipe book button. ♥
Thanks in advance

![MissingButton](https://user-images.githubusercontent.com/110115789/210052622-e6c51d81-61ea-405f-ab55-c5e009f92d7d.png)

Picture from 1.18 version where things are still "normal".

### Mod suggestion

I honestly cant tell which mod is at fault for this.

**Comment by Saereth** [comment]
it is intentional, it's fastbench, you can turn it back on in the config if you like config/fastbench.cfg
```

general {
    # If the crafting table has a tooltip [default: false]
    B:"Disable tooltip on crafting table"=false

    # If the recipe book button is removed. [default: true]
    B:"Remove Recipe Book Button"=true
}


```
set the "remove recipe book button" to false
